### PR TITLE
Add docs for updating and deleting an LDAP connection

### PR DESCRIPTION
**Pull request by harry-kim**
- [x]  Set milestone label in pull request (SDE version)
- [x]  Updated changelog.md
- [x]  Completed documentation for the new endpoint.
- [x]  Feature has been merged into SDE develop.

Refs: #SDE-3432

### CODE REVIEW

**Comment by funkaoshi** [comment]
You need to do the following:

- [ ]  Set milestone label in pull request (SDE version)
- [ ]  Updated changelog.md
- [ ]  Completed documentation for the new endpoint.
- [ ]  Feature has been merged into SDE develop.

This is supposed to show up in the template when you make an MR. Weird.

File: source/includes/connections.md [hunk 1]
```diff
@@ -231,6 +231,137 @@ base_dn             | No       | The base dn (will be computed from bind_dn if u
 
 ---
 
+## Delete an LDAP Connection
+
+```http
+DELETE /api/v2/connections/ldap/ HTTP/1.1
```
Comment (RouganStriker) [inline source/includes/connections.md]: DELETE /api/v2/connections/ldap/{connection_id}

File: source/includes/connections.md [hunk 2]
```diff
@@ -231,6 +231,137 @@ base_dn             | No       | The base dn (will be computed from bind_dn if u
...
+```http
+PUT /api/v2/connections/ldap/31/ HTTP/1.1
+Accept: application/json
+Authorization: Token "YOUR SDE ACCESS TOKEN"
+
+{
+  "alias": "Example Connection Updated Name",
+  "system": "LDAP",
+  "frequency": "daily",
+  "command": "sync_ldap",
```
Comment (RouganStriker) [inline source/includes/connections.md]: Can you remove the non edittable fields from put? like command and system

**Comment by aeldamaty** [comment]
+1

File: source/includes/connections.md (hunk 1 above)
Comment (harry-kim) [inline source/includes/connections.md]: updated

File: source/includes/connections.md (hunk 2 above)
Comment (harry-kim) [inline source/includes/connections.md]: updated

File: source/includes/changelog.md [hunk 3]
```diff
@@ -3,6 +3,7 @@
 ## v4.4
 
 - Added include field `last_job` for connections returned by the ALM, Analysis and LDAP Connection endpoints.
+- Added documentation for LDAP connections and jobs
```
Comment (harry-kim) [inline source/includes/changelog.md]: is this okay?

Comment (funkaoshi) [inline source/includes/changelog.md]: You should say what you can now do with the endpoint. So something like "you can create, edit and delete LDAP connections" or something. Look at the other examples. We could probably word them all better, perhaps.

Comment (harry-kim) [inline source/includes/changelog.md]: updated

**Pull request merged**